.gitignore

```diff
@@ -24,6 +24,7 @@ local.sbt
 .idea/
 out/
 .d
+*~
```

Comment:
This line is unneeded. You should put vim/gedit temp file in your global .gitignore or prevent the editor spamming your folder.